sources/packages/web/src/components/common/StudentOverawardDetails.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+  <v-card class="mb-5">
+    <v-container>
+      <body-header
+        title="Overaward balances"
+        subTitle="A balance of overawards broken down by award type"
+      />
+      <v-row>
+        <v-col>
+          <title-value
+            :propertyValue="
+              formatCurrency(overawardBalance.overawardBalanceValues?.CSLF, '')
```

Comment:
will `overawardBalanceValues` be undefined in any case? `formatCurrency` first parameter is `number | string`, its not expecting `undefined`.
![image](https://user-images.githubusercontent.com/77353155/221006627-3f7b208c-99ec-4e74-8a27-6577c7af3a45.png)

Reply:
`formatCurrency` `moneyAmount` parameter can be changed to accept `undefined`.

Reply:
If it is expecting undefined, then it will be like an optional parameter right? it's consumers may use it like formatCurrency().

IMO like it as is will be better.